src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisProviderTest.java

```diff
@@ -105,4 +109,25 @@ void jsonTest() throws IOException {
         parameters.addExtension(OpenSecurityAnalysisParameters.class, parametersExt);
         roundTripTest(parameters, JsonSecurityAnalysisParameters::write, JsonSecurityAnalysisParameters::read, "/sa-params.json");
     }
+
+    @Test
+    void testContingencyParametersExtension() {
```

Comment:
Renaming of ContingencyParameters into ContingencyLfParameters seems validated.
So also rename tests accordingly to testContingencyLfParametersExtension and testContingencyLfParametersExtensionJson

Reply:
fixed in a8f0dabbdba0b462b430a614e05b568d1ce58a14